example/10-secret-certificate-management-config.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
As discussed: Please add this to the Gardener Helm chart so that it is not needed to be installed/deployed separately.